src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -32,28 +32,22 @@
 
 DATA_SAVE_ID  = wx.NewId()
 
-
 class ExtendedToolbar(NavToolbar):
     """An extended plotting toolbar with a save and close button."""
 
     def __init__(self, canvas):
+        # override class var to exclude subplots
+        self.toolitems = tuple(el for el in NavToolbar.toolitems\
+                if el[0] != 'Subplots')
         NavToolbar.__init__(self, canvas)
-        # Get rid of the configure subplots button
-        if hasattr(self, 'wx_ids'):
-            self.DeleteTool(self.wx_ids['Subplots'])
-        else:
-            self.DeleteToolByPos(6)
         # Add new buttons
         self.AddSimpleTool(wx.ID_PRINT,
                wx.ArtProvider.GetBitmap(wx.ART_PRINT, wx.ART_TOOLBAR),
                'Print', 'print graph')
         self.AddSimpleTool(DATA_SAVE_ID,
-               _load_bitmap('stock_save_as.xpm'),
+               _load_bitmap('filesave.png'),
```

Comment:
yeah, I just finished this modification, but I encountered something weird. ``iconpath`` actually return a path as:
````
/repo/location/diffpy.pdfgui/src/icons/*
```` 
but the ``icons/`` is actually at the same level as ``src/`` in this repo. 

So by running ``python -m diffpy.pdfgui.gui.extendedplotframe``, I would have following error:
````
12:50:42: can't open file '/home/timothy/Documents/Repo/diffpy.pdfgui/src/icons/stock_save_as_mpl2.png' (error 2: No such file or directory)
12:50:42: Failed to load image from file "/home/timothy/Documents/Repo/diffpy.pdfgui/src/icons/stock_save_as_mpl2.png".
````
that seems to be the same error when I run ``python.app -m diffpy.pdfgui.applications.pdfgui
`` as mentioned in #15.

Does it come from my local configuration?

Also, since we might have to prepare two files, is ``matplotlib.__version__`` a good approach to check?

Thanks for that comments, they are all very helpful!